### DISCUSSION TITLE
距离我上篇笔记还是在? - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#197] [General]
# 距离我上篇笔记还是在? - 愧怍的小站

当我起这个标题时，其实我已经很久没更新（翻看）过笔记了，甚至我都不记得我的博客还有笔记这个东西。

https://kuizuo.cn/why-i-dont-write-notes#comments

**Comment by makerjackie** [comment]
说的很棒!

**Comment by Elite-zx** [comment]
博主说的我深表赞同。我个人认为，初学一个知识点的时候记笔记，真的没什么作用。最多就是用自己话把书上的东西重新解释一遍，还不一定准确。有这功夫为什么不去做点实际的东西练练手呢？当你在实践中用多了这些知识点，自然才有更深刻的体会和理解，这个时候把这种体会和理解写下来，这样的文字内容才稍微有点意义

**Reply by kuizuo** [reply]
不过我现在记录也比较少了，平常忙于开发，在实现完功能之后总结的想法反而少了很多。

现在写 blog 更多只是为了记录，而不是分享了
